docker-compose.yml

```diff
@@ -45,20 +55,25 @@ services:
       MOCK_ACCOUNT_PASSWORD:
 
   ui:
-    build: ./src/ui
-    image: exip/src/ui
+    build:
+      context: ./src/ui
+      args:
+        PORT: ${UI_PORT}
+    image: ui
+    container_name: exip_ui
     restart: always
     depends_on:
       - database
       - api
     ports:
-      - "5000:5000"
+      - "${UI_PORT}:${UI_PORT}"
     volumes:
       - ./src/ui/server:/app/server:ro
       - ./src/ui/public:/app/public:ro
       - ./src/ui/templates:/app/templates:ro
     environment:
-      PORT: 5000
+      PORT: ${UI_PORT}
+      NODE_ENV:
```

Comment:
we could probably populate these `NODE_ENV` with `development` value by default since we only use it locally for development

Reply:
We use `NODE_ENV` with in the source code whether to populate HTTP based authentication. This is effects healthcheck if set to `development` `ui` healtcheck will fail as it will return 404 instead of 2022.